lib/tasks/after_tasks.rake

```diff
@@ -694,16 +694,6 @@ namespace :After do
       REDIS_GENERAL.del(key)
     end
   end
-
-  desc "Copy anon_commenting_disabled to comment_permissions."
```

Comment:
We keep `add_user_id_to_kudos` around in case people need to update their development database, although in that case, without running the task before the pseud_id removal migration, you would lose all logged-in user kudos data. This task is not as necessary.

I'm okay with removing this task, but I could be convinced otherwise.

Reply:
I don't have strong feelings either way. Maybe I should just add it back?

Reply:
Yeah, let's add it back for now.